packages/preview/quiztime/0.1.0/README.md

```diff
@@ -0,0 +1,35 @@
+# Quiztime
+
+Typst package to create quizzes.
+
+```typst
+#import "@preview/quiztime:0.1.0": question, quiz
+
+#let q1 = question(
+  text: "What color is the sky?",
+  kind: "multiple-choice",
+  answers: ("Blue", "Red", "Yellow", "Gray"),
+)
+
+#let q2 = question(
+  text: "Why do you think is the world where it is?",
+  kind: "text",
+  answers: ("Because no one took proper care of it.",),
+  show-solutions: true,
+  answer-space: 50pt,
+)
+
+#let q3 = question(
+  text: "What color is the sea?",
+  kind: "multiple-choice",
+  answers: ("Blue", "Red", "Yellow", "Gray"),
+  true-answers: (1,),
+  show-solutions: true
+)
+
+#quiz(questions: (q1,q2,q3), pre-numbering: "Question ")
+```
+
+Gives a list of quiz questions that are multiple-choice or text-based (i.e., space for writing freely). Solutions may be enabled per question.
+
+![](./output.png)
```

Comment:
Sorry if I wasn't clear, but I think it would be better to have the image description in the dedicated place (between the `[]`). Otherwise, screen readers would just say something like "Image" when describing the picture, and people on poor internet connections would see a "broken picture" icon.

Reply:
I see, valid! I added a description.